src/vaults/vaultDataService.ts

```diff
@@ -50,3 +52,21 @@ export async function getRecentDailyCollateral(
         throw e;
     }
 }
+
+export async function getAllVaults(): Promise<VaultData[]> {
+    try {
+        const res = await pool.query(`
+            SELECT DISTINCT ON (reg.vault_id)
+                reg.vault_id,
+                reg.collateral
```

Comment:
This will return the collateral the vault registered with, but not necessarily the vault's current collateral, since it can lock more or withdraw some.
I suggest either
1. Removing the collateral column, and renaming to "getVaultsList" or similar; or
2. Expanding the query to calculate the current collateral (using either collateral `Lock` and `Unlock` events, or vault `LockAdditionalCollateral` and `WithdrawCollateral` events - not sure which is best) - this would make the query also useful for the dashboard; or
3. Renaming to "getVaultRegistrations" to make it clear it only has data from the registration event.